app/models/book_author.rb

```diff
@@ -0,0 +1,4 @@
+class BookAuthor < ActiveRecord::Base
```

Comment:
That's fine! At first glance it just looked like premature optimization.

What do you think about renaming the model to "Credit" to make more sense from a vocabulary standpoint?